test/nodejs-bundle/circuit/dial-over-specific-relay-transport.js

```diff
@@ -0,0 +1,141 @@
+/* eslint-env mocha */
+'use strict'
+
+const TCP = require('libp2p-tcp')
+const WS = require('libp2p-websockets')
+const multiplex = require('libp2p-multiplex')
+const multiaddr = require('multiaddr')
+
+const waterfall = require('async/waterfall')
+const utils = require('./helpers/utils')
+
+const sinon = require('sinon')
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+
+const expect = chai.expect
+
+const nodeKeys = require('./fixtures/nodes')
+const Circuit = require('libp2p-circuit')
+
+describe(`listen on an explicit chained relay addr`, function () {
+  this.timeout(500000)
```

Comment:
This timeout is not ok. @dryajov please figure out why it needs such a timeout and fix or document why it needs to be here